client/my-sites/checkout/checkout-thank-you/atomic-store-thank-you-card.jsx

```diff
@@ -25,7 +25,7 @@ class AtomicStoreThankYouCard extends Component {
 			<div className="checkout-thank-you__atomic-store-action-buttons">
 				<a
 					className={ classNames( 'button', 'thank-you-card__button' ) }
-					href={ `/store/${ site.slug }` }
+					href={ site.URL + '/wp-admin/admin.php?page=wc-setup' }
```

Comment:
Just found out that the path is `/wp-admin/admin.php?page=wc-setup-checklist&calypsoify=1`
but that it's not in yet. `wc-setup` works for now, just a heads up that this will change